tsconfig.json

```
@@ -10,5 +10,5 @@
 		"strict": true,
 		"target": "ES2021"
 	},
-	"include": ["src"]
+	"include": ["src", "typings"]
```

We shouldn't need this anymore.

```suggestion
	"include": ["src"]
```